docs/http-api.md

```diff
@@ -163,6 +163,26 @@ POST /metrics/delete
 curl -H "X-Org-Id: 12345" --data query=statsd.fakesite.counters.session_start.*.count "http://localhost:6060/metrics/delete"
 ```
 
+
+## Deleting tagged metrics
+
+This will delete the metrics (technically metricdefinitions) matching the `path` parameter(s).
+Note that the data stays in the datastore until it expires.
+Should the metrics enter the system again with the same metadata, the data will show up again.
+
+```
+POST /tags/delSeries
+```
+
+* header `X-Org-Id` required
+* path (required, multiple allowed): A single Graphite series
+
+#### Example
+
+```bash
+curl -H "X-Org-Id: 12345" -d path=some.series;key=value -d path=another.series;tag=value "http://localhost:6060/tags/delSeries"
```

Comment:
wouldn't the arguments to `-d` have to be quoted? otherwise bash is going to cut them off at `;` or not?

I tried by starting `nc -l -p 8080` in one terminal and then:

```
curl 'http://localhost:8080/abc' -d path=a.b.c;tag=value
```

gives 

```
replay@mst-nb:~/go/src/github.com/grafana/metrictank$ nc -l -p 8080
POST /abc HTTP/1.1
Host: localhost:8080
User-Agent: curl/7.58.0
Accept: */*
Content-Length: 10
Content-Type: application/x-www-form-urlencoded

path=a.b.c
```

Reply:
good catch: 94a810c92ef3eb15e556bae7df68a3680f54b713